packages/frontend/src/hooks/thirdPartyServices/useSentry.ts

```diff
@@ -15,8 +14,18 @@ const useSentry = (
                 dsn: sentryConfig.dsn,
                 release: sentryConfig.release,
                 environment: sentryConfig.environment,
-                integrations: [new Integrations.BrowserTracing()],
-                tracesSampleRate: 0.2,
+                integrations: [
+                    Sentry.browserTracingIntegration(),
+                    Sentry.replayIntegration(),
+                ],
+                tracesSampler(samplingContext) {
+                    // TODO: verify if this is the right way to sample errors
+                    if (samplingContext.transactionContext?.name === 'error') {
+                        return 1.0;
+                    }
+                    return 0.5;
+                },
+                replaysOnErrorSampleRate: 1.0,
```

Comment:
`replaysSessionSampleRate`'s default value is `0` so that can be left unreferenced here.
source: https://docs.sentry.io/platforms/javascript/session-replay/configuration/#general-integration-configuration